aspnetcore/security/authentication/2fa.md

```diff
@@ -21,7 +21,7 @@ uid: security/authentication/2fa
 
 By [Rick Anderson](https://twitter.com/RickAndMSFT)
```

Comment:
Can you please be more specific regarding your statement "creating a new project"? We don't know exactly what to do.